.changeset/neat-wolves-wink.md

```diff
@@ -0,0 +1,5 @@
+---
+'@redux-saga/core': patch
```

Comment:
Since `redux-saga` is a more user-facing package - let's add this to its CHANGELOG too

```suggestion
'redux-saga': patch
'@redux-saga/core': patch
```